Animations/Zoom/DownBig.js

```diff
@@ -26,5 +28,5 @@ var ZoomDownBig = {
     'transform': 'scale(.1) translateY(-2000px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('ZoomDownBig', ZoomDownBig);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('ZoomDownBig', ZoomDownBig);
```

Comment:
This does not need to have `'ZoomDownBig'` passed into it since we have defined a `className` property in `ZoomDownBig`.